trismart_smartboard_integration/models/smartboard_request.py

```diff
@@ -0,0 +1,68 @@
+from odoo import models, fields, api, modules
+from odoo.exceptions import UserError
+from ..utils.endpoint import SmartBoardAPIURL
+from ..utils.exceptions import HTTPError
+import requests
+import json
+
+
+class SmartBoardRequest(models.TransientModel):
+    _name = 'smartboard.request'
+    _description = 'SmartBoard Request'
+
+    def smartboard_api_request(self, url, payload):
+        """
+        Calling SmartBoard API
+        :param url:
+        :param payload:
+        :return:
+        """
+        smartboard_endpoint = SmartBoardAPIURL.SMARTBOARD_URL_ENDPOINT
```

Comment:
@lamnguyen-novobi: We can have a setting to toggle between dev and production mode and load the URL endpoint accordingly or let user enter the URL endpoint. As a result, we don't need to update the source code when moving to production.